src/main/java/za/ac/cput/controller/EmployeeController.java

```diff
@@ -39,17 +39,17 @@ public ResponseEntity<Employee> read(@PathVariable String employee_id){
         return ResponseEntity.ok(response);
     }
 
-    @DeleteMapping("delete/{employee}")
-    public ResponseEntity<Void> delete(@PathVariable Employee employee){
-        log.info("Delete request: {}", employee);
-        this.employeeAPI.delete(employee);
-        return ResponseEntity.noContent().build();
-    }
-
     @GetMapping("all")
     public ResponseEntity<List<Employee>> getAll(){
         List<Employee> response = this.employeeAPI.getAll();
         return ResponseEntity.ok(response);
     }
 
+    @DeleteMapping("delete")
```

Comment:
Please revert this change the delete method works as intended currently and doesn't need this change